integration/setup/common.go

```diff
@@ -108,11 +116,16 @@ func SkipForPostgresWithReason(tb testing.TB, reason string) {
 func checkMongoDBURI(tb testing.TB, ctx context.Context, uri string) bool {
 	tb.Helper()
 
+	_, span := otel.Tracer("").Start(ctx, "checkMongoDBURI")
```

Comment:
What does `Tracer("")` do? A bit strange to have empty tracer arg. Example has name space in it but I'm not sure the impact for reporting https://github.com/open-telemetry/opentelemetry-go/blob/main/example/namedtracer/foo/foo.go#L34

Reply:
It will use the default name for tracer https://github.com/open-telemetry/opentelemetry-go/blob/c0fb8dec4cf2183f852ebd7e663053614e044bf4/trace.go#L23

Reply:
@chilagrow Dmitry mentioned the exact reason. I suggest not using named tracers unless we really need them. If it solves the question, could you resolve the thread, please?